src/alfasim_score/converter/alfacase/convert_alfacase.py

```diff
@@ -0,0 +1,49 @@
+from alfasim_sdk import AnnulusDescription
+from alfasim_sdk import FormationDescription
+from alfasim_sdk import ProfileDescription
+from alfasim_sdk import WellDescription
+from alfasim_sdk import XAndYDescription
+from barril.units import Scalar
+
+from alfasim_score.constants import ANNULUS_TOP_NODE_NAME
+from alfasim_score.constants import WELLBORE_BOTTOM_NODE
+from alfasim_score.constants import WELLBORE_TOP_NODE
+from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
+from alfasim_score.units import LENGTH_UNIT
+
+
+class AlfacaseConverter:
+    def __init__(self, score_reader: ScoreInputReader):
+        self.score_input = score_reader
+        self.well_name = score_reader.score_filepath.stem
+
+    def _get_well_start_vertical_position(self) -> Scalar:
+        _, y = self.score_input.read_well_trajectory()
+        return Scalar(y[0], LENGTH_UNIT)
+
+    def _convert_well_trajectory(self) -> ProfileDescription:
+        """
+        Convert the trajectory for the imported well.
+        NOTE: all positions don't start to count as zero at ANM, but they use the same values
+        from the input SCORE file.
+        """
+        x, y = self.score_input.read_well_trajectory()
```

Comment:
I tested the trajectory in ALFAsim and it seems to work well.
However, in formation and environment, the reference y coordinate must be ZERO.

Reply:
Ok, got it.